riptable/rt_datetime.py

```diff
@@ -542,6 +542,14 @@ def __new__(cls, values):
         instance._display_length = DisplayLength.Long
         return instance
 
+    # ------------------------------------------------------------
+    def __array_finalize__(self, obj):
+        if obj is None:
+            return
+        from_peer = isinstance(obj, DateTimeBase)
+        self._display_length = obj._display_length if from_peer else DisplayLength.Long
+        self._timezone = obj._timezone if from_peer else 'GMT'

```

Comment:
I think ``_timezone`` attribute is an instance of ``rt.TimeZone``; but here, if ``from_peer`` is False, you assign a string to the attribute, which would cause things to break later.

I think it'd also be better to use ``'UTC'`` instead of ``'GMT'`` as the default timezone here -- that's the current behavior of ``DateTimeNano.__new__()``.

You can fix both of these by changing this line to:
```
self._timezone = obj._timezone if from_peer else TypeRegister.TimeZone(from_tz='UTC', to_tz='UTC')
```

Reply:
Agreed